File: src/lessons/welcome/methods/picture/PictureMono1Entity.c
//RemoteBuggle

void mark();
void makeV();

/* BEGIN TEMPLATE */
void run(){
	/* BEGIN SOLUTION */
	makeV();
	makeV();
	makeV();
	makeV();
}

void mark() {
	brushDown();
	brushUp();
}

void makeV() {
	forward(2);
	mark();

	forward(1);
	left();
	forward(1);
	mark();

	backward(1);
	right();
	forward(1);
	mark();

	forward(2);
	left();
/* END SOLUTION */

}
/* END TEMPLATE */

File: src/lessons/welcome/methods/picture/PictureMono1Entity.scala
package lessons.welcome.methods.picture;

import java.awt.Color;

import plm.universe.bugglequest.SimpleBuggle;
class PictureMono1Entity extends SimpleBuggle {

	/* BEGIN TEMPLATE */
	override def run() {
		/* BEGIN SOLUTION */
		for (i <- 1 to 4) {
			makeV();
		}
	}
	def mark() {
		brushDown();
		brushUp();
	}

	def makeV() {
		forward(2);
		mark();

		stepForward();
		left();
		stepForward();
		mark();

		stepBackward();
		right();
		stepForward();
		mark();

		forward(2);
		left();
		/* END SOLUTION */
	}
	/* END TEMPLATE */
}
